lib/modules/versioning/hermit/index.spec.ts

```diff
@@ -0,0 +1,210 @@
+import { HermitVersioning } from './index';
+
+describe('modules/versioning/hermit/index', () => {
+  const versioning = new HermitVersioning();
+
+  describe('isStable', () => {
+    test.each`
+      version      | expected
+      ${'1'}       | ${true}
+      ${'1.2'}     | ${true}
+      ${'@1'}      | ${false}
+      ${'@1.2'}    | ${false}
+      ${'@1.2.3'}  | ${false}
+      ${'@latest'} | ${false}
+      ${'@stable'} | ${false}
+    `('isStable("$version") === $expected', ({ version, expected }) => {
```

Comment:
```suggestion
    `('("$version") === $expected', ({ version, expected }) => {
```

we don't need to duplicate the function name, parent describe is prepended by jest.